sstables/sstable_directory.hh

```diff
@@ -51,10 +52,14 @@ public:
     requires std::is_invocable_r_v<future<>, Func, typename std::ranges::range_value_t<Container>&>
     future<> parallel_for_each(Container& c, Func func) {
         co_await max_concurrent_for_each(c, _concurrency, [&] (auto& el) -> future<>{
-            auto units = co_await get_units(_sem, 1);
```

Comment:
I have no idea how the original call (probably) found `seastar::get_units`
:thinking:

Reply:
> I have no idea how the original call (probably) found `seastar::get_units` 🤔

`#include "seastarx.hh"` has
```
using namespace seastar;
```

Reply:
> #include "seastarx.hh"

ah ha!